hugo/content/downloads.md

```diff
@@ -64,22 +64,22 @@ document.getElementById('suggested-download').innerHTML = getPlatformContent(get
 ## Manual Download
 
 {{< content-layout/downloads >}}
-{{< content-layout/download name="Windows 64-bit" href="windows-64" osclass="windows">}}
-{{< content-layout/download name="Windows 32-bit" href="windows-32" osclass="windows">}}
+{{< content-layout/download name="Windows (x64)" href="https://dl.mumble.info/latest/stable/client-windows-x64" osclass="windows">}}
+{{< content-layout/download name="Windows (x86)" href="https://dl.mumble.info/latest/stable/client-windows-x86" osclass="windows">}}
```

Comment:
I prefer "64-bit" and "32-bit" over "x64" and "x86"

Reply:
It's ambiguous.

64-bit could refer to either x64 or ARM64.
32-bit could refer to either x86 or ARM.

Reply:
Ah yes indeed